### PR TITLE
DOC:fix README typos

### PR DESCRIPTION
**Pull request by anonmouse1**
Two typo updates to README file:

- Line 41: while `Remote*` factories represent _that_ kinds of records we get from an API or external package >> while `Remote*` factories represent _the_ kinds of records we get from an API or external package.

- Line 110: The current expectation is that _you we tag_ to freeze the code >> The current expectation is that _you make a git tag_ to freeze the code

Note:some other formatting changes made throughout through use of Prettier extension(eg.remove whitespace between parenthesis) which can be ignored for this PR if desired

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 ## Requirements
 
-* Xcode 13 or above
-* [Android and iOS environment setup](https://reactnative.dev/docs/environment-setup) described in the RN docs
+- Xcode 13 or above
+- [Android and iOS environment setup](https://reactnative.dev/docs/environment-setup) described in the RN docs
```
Comment (kueda) [inline README.md:6]: `*` and `-` are [both supported](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#lists) for lists, so this change isn't necessary.

File: README.md [hunk 2]
```diff
@@ -92,7 +92,6 @@ git commit -a -m "Updated translations"
 1. Run `react-native clean-project`. This will give you options to clean caches, clean builds, reinstall pods, and reinstall node_modules. Using this eliminates a lot of hard-to-diagnose build issues.
 1. If you're running on an M series chip, you may need to install a specific version of NDK to the app to build for Android. See `android/build.gradle`
 
-
```
Comment (kueda) [inline README.md:95]: Please remove this and the other unnecessary whitespace additions.

File: README.md [hunk 3]
```diff
@@ -49,28 +49,28 @@ We're using [Jest](https://jestjs.io/) and [React Native Testing Library](https:
...
-    };
-    ````
+   ```jsx
+   import { useTranslation } from "react-i18next";
+   const MyComponent = () => {
+     const { t } = useTranslation();
+     return (
+       <View>
+         <Text>{t("ABOUT-PROJECTS")}</Text>
+         <Text>{t("projects-description")}</Text>
```
Comment (kueda) [inline README.md:69]: Our convention is to have spaces inside parens, so please revert these two lines to the way the were. The extra \` is a good catch, though.

**Review by kueda — CHANGES_REQUESTED** [review]
Thanks for the typo fixes, but please try to leave the rest of it the way it was.

**Comment by kueda** [comment]
@anonmouse1 were you planning to keep working on this? If so I'll leave it open.

File: README.md (hunk 3 above)
Comment (anonmouse1) [reply README.md:69]: @kueda you can close this PR as I'd like to do it again from a fresh codebase as I encountered issues when trying to push my changes after others were already merged.
